statsmodels/regression/tests/test_robustcov.py

```diff
@@ -33,7 +34,32 @@ def test_basic(self):
         assert_allclose(self.bse_robust, res2.bse, rtol=rtol)
         assert_allclose(self.cov_robust, res2.cov, rtol=rtol)
 
-    def test_tests(self):
+    @pytest.mark.smoke
+    def test_t_test_summary(self):
+        res1 = self.res1
+        mat = np.eye(len(res1.params))
+        # TODO: if the t_test call is expensive, possibly make it a fixture?
+        tt = res1.t_test(mat, cov_p=self.cov_robust)
+
+        tt.summary()
+
+    @pytest.mark.smoke
+    def test_t_test_summary_frame(self):
```

Comment:
we are not splitting those
the collection of tests is driven by grouping topic and reuse of existing results.
i.e. as I mentioned a few times before, I want to keep the test methods larger instead of getting 20 (?) of them in one class.

Reply:
Forget it.  Closing.

Reply:
Splitting helps to fix problems.  If you have a failure in say the last part of a long test, then you have to rerun the entire test (if slow) to get debug one part.  This can involve repeated hitting debugging points and continuing to get to the actual problem.